builtin/providers/aws/resource_aws_autoscaling_scaling_policy_test.go

```diff
@@ -0,0 +1,125 @@
+package aws
+
+import(
+    "fmt"
+    //"math/rand"
+    //"strings"
+    "testing"
+    //"time"
+
+    "github.com/awslabs/aws-sdk-go/aws"
+    "github.com/awslabs/aws-sdk-go/service/autoscaling"
+    "github.com/hashicorp/terraform/helper/resource"
+    "github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSAutoscalingScalingPolicy_basic(t *testing.T) {
+    var policy autoscaling.ScalingPolicy
```

Comment:
I have `testAccCheckScalingPolicyExists` defined which accepts `policy` and checks it's existence.